Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+node {
+  currentBuild.result = "SUCCESS"
+
+  try {
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Setup') {
+      sh 'docker system prune --force'
+      sh 'docker build -f docker-base.Dockerfile -t docker-base -t docker-base:latest .'
+    }
+
+    stage('Build Platform Deploy Docker') {
+      env.NODE_ENV = "production"
+      sh 'aws s3 cp s3://citizens-config/env-platform-deploy.sh env-platform-deploy.sh'
+      sh 'docker build -f platform-deploy.Dockerfile -t platform-deploy -t 409163884834.dkr.ecr.us-east-2.amazonaws.com/platform-deploy:$BUILD_NUMBER -t 409163884834.dkr.ecr.us-east-2.amazonaws.com/platform-deploy:latest .'
+    }
+
+    stage('Build Web Auth Docker') {
+      env.NODE_ENV = "production"
+      sh 'aws s3 cp s3://citizens-config/env-web-auth.sh env-web-auth.sh'
+      sh 'docker build -f web-auth.Dockerfile -t web-auth -t 409163884834.dkr.ecr.us-east-2.amazonaws.com/web-auth:$BUILD_NUMBER -t 409163884834.dkr.ecr.us-east-2.amazonaws.com/web-auth:latest .'
+    }
+
+    stage('Build Accounts') {
+      sh 'rm -rf ./packages/app-account/build/'
+      nodejs(nodeJSInstallationName: '10.4.1') {
+        sh 'npm install'
+        sh 'export APP_NAME=app-account; npm run postinstall; npm run build'
+      }
+    }
+
+    stage('Deploy Platform Deploy') {
```

Comment:
These Deploy steps should be tweaked to only run in master if this is going to be merged.